order/buyer/receive/single-delivery-order-event.md

```diff
@@ -6,13 +6,13 @@ description: How to receive a simple order response sent by the supplier
 
 Tradecloud will send a purchase order response to the buyer when an order event has been triggered.
```

Comment:
Note: the page description still says:

```
description: How to receive a simple order response sent by the supplier
```

Also reword "simple" to "single" here.

Reply:
fixed